### DISCUSSION TITLE
Trying to get an answer from Eugen is like pulling teeth

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#23620] [General]
A week ago I emailed him about the server order change and to reclassify our instance as general, no response. I sent a message to him directly on Mastodon, no response. 

If this is what’s to expect from the main source of where people find servers to join then I can see why people think mastodon isn’t a serious contender for social media.

**Comment by markcellus** [comment]
Sorry you're not getting your request answered as quickly as you hoped. I'm sure Eugen gets a _ton_ of messages from people, so I'd imagine it's pretty tough to respond to everyone. Did you try emailing him or the team directly?

Also, its good to keep in mind that Mastodon is open-sourced. Not only is it not trying to be any contender for social media, the maintainers are in no way obligated to fulfill anybody's request or even respond to anyone if they don't want. I'm not saying they shouldn't. But it would be wise to adjust your expectations, given that this is open-source software and the amount of requests they get that may be similar to yours.

**Reply by ghost** [reply]
The email wasn’t sent to him directly, it was to a generic address as requested by the website. 

I understand this is open source, but when you’re managing joinmastodon.org and it’s the primary website of where people go to find servers to sign up you have an obligation to be contactable, they get paid by sponsors/people so it’s not like this is all being done out of the kindness of their hearts. If I had waited a week for any other service to respond I’d have moved on. 

I know where which action I’ll be taking, and that’s giving my 30k of users 7 days to find a new home. Mastodon server covenant be damned.
